### DISCUSSION TITLE
Feature request: turn off storing content or change the bookmark links

### DISCUSSION THREAD

**Discussion by a1e7de48f89e** [go-shiori/shiori#335] [Feature requests]
Somewhat related to https://github.com/go-shiori/shiori/issues/203, see my last comment there in particular.

It would be great to have an option to either turn off storing a copy of the plaintext content of a page altogether (the one that is apparently always stored if possible, not the archive, which is optional already) or to change the links on the bookmarks so the image and main title will always link to the original URL.

I think the way it's currently linked (the image and main title linking to the stored content/archive) is not ideal especially on mobile devices where clicking the original URL below is a bit of a pain. I would rather just link the two icons to stored content/archive respectively and everything else to the original URL. It's usually my intent to view the original, unless it's no longer available, in which case I'll fall back to the stored content/archive. This could also be optional, I guess, if people prefer it the way it currently is.

**Comment by deanishe** [comment]
> change the links on the bookmarks so the image and main title will always link to the original URL.

Probably this. Or make the original link easier to click/tap.
